static/app/gettingStartedDocs/javascript/nuxt.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import {renderWithOnboardingLayout} from 'sentry-test/onboarding/renderWithOnboardingLayout';
+import {screen} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import {ProductSolution} from 'sentry/components/onboarding/productSelection';
+
+import docs from './nuxt';
+
+describe('javascript-nuxt onboarding docs', function () {
+  it('renders onboarding docs correctly', () => {
+    renderWithOnboardingLayout(docs);
+
+    // Renders main headings
+    expect(screen.getByRole('heading', {name: 'Install'})).toBeInTheDocument();
+    expect(screen.getByRole('heading', {name: 'Configure SDK'})).toBeInTheDocument();
+    expect(screen.getByRole('heading', {name: 'Upload Source Maps'})).toBeInTheDocument();
+    expect(screen.getByRole('heading', {name: 'Verify'})).toBeInTheDocument();
+
+    // Includes 2 import statements
+    expect(
+      screen.getAllByText(
+        textWithMarkupMatcher(/import \* as Sentry from "@sentry\/nuxt"/)
+      )
+    ).toHaveLength(2);
+  });
+
+  it('displays sample rates by default', () => {
+    renderWithOnboardingLayout(docs, {
+      selectedProducts: [
+        ProductSolution.ERROR_MONITORING,
+        ProductSolution.PERFORMANCE_MONITORING,
+        ProductSolution.SESSION_REPLAY,
+      ],
+    });
+
+    expect(screen.queryAllByText(textWithMarkupMatcher(/tracesSampleRate/))).toHaveLength(
+      2
+    ); // client and server
+    expect(
+      screen.queryByText(textWithMarkupMatcher(/replaysSessionSampleRate/))
```

Comment:
```suggestion
      screen.getByText(textWithMarkupMatcher(/replaysSessionSampleRate/))
```

`queryBy` is usually used when we do not expect the dom to have the element 🙂